src/components/Footer/style.css

```diff
@@ -0,0 +1,29 @@
+#h1{
+    font-style: "Quicksand";
+    font-size: 64px;
+    text-align: center;
+    margin-top: 0;
+    color: #000000;
+}
+
+#icon-box{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+}
+
+.icon{
+    margin-left: 2.3rem;
+    margin-right: 2.3rem;
+}
+
+img{
+    width: 7.8rem;
+    height: 7.8rem;
```

Comment:
these feel really big to me, i think 4rem or 5rem might be better. you can also post in #design and see what they think